src/Tizen.Applications.Common/Tizen.Applications/TizenSynchronizationContext.cs

```diff
@@ -115,7 +125,7 @@ private void Post(Action action)
                 id = _transactionId++;
             }
             _handlerMap.TryAdd(id, action);
-            Interop.Glib.IdleAdd(_wrapperHandler, (IntPtr)id);
+            _source = Interop.Glib.IdleAdd(_wrapperHandler, (IntPtr)id);
```

Comment:
I think Post() could be called multiple times, and _source could be overwritten. Could you check it?